processes/third_party_vuln_process.md

```diff
@@ -0,0 +1,88 @@
+This document describes the management of vulnerabilities within the Node.js
+ecosystem. Vulnerabilities in Node.js core are out of this scope.
+
+# Definitions
+
+* package: in this document, a package is a module available widely to use 
```

Comment:
I'd suggest 

a package available for use with Node.js and hosted on the npmjs.org repository.